pyasic/miners/backends/epic.py

```diff
@@ -114,6 +115,52 @@ async def get_config(self) -> MinerConfig:
         self.config = cfg
         return self.config
 
+    async def send_config(self, config: MinerConfig, user_suffix: str = None) -> None:
+        self.config = config
+        conf = self.config.as_epic(user_suffix=user_suffix)
+
+        try:
+            # Temps
+            data = await self.web.set_shutdown_temp(conf["temp_control"]["hot_temp"])
+            # Fans
+            if conf["fan_control"]["mode"] == "Manual":
+                data = await self.web.set_fan(
+                    {"Manual": int(conf["fan_control"]["speed"])}
+                )
+            if conf["fan_control"]["mode"] == "Auto":
+                data = await self.web.set_fan(
+                    {
+                        "Auto": {
+                            "Idle Speed": int(conf["fan_control"]["speed"]),
+                            "Target Temperature": int(
+                                conf["temp_control"]["target_temp"]
+                            ),
+                        }
+                    }
+                )
+
+            ## Mining Mode -- Need to handle that you may not be able to change while miner is tuning
+            if conf["ptune"]["enabled"] == True:
+                data = await self.web.set_ptune_enable(True)
+                data = await self.web.set_ptune_algo(
+                    {"algo": conf["ptune"]["algo"], "target": conf["ptune"]["target"]}
```

Comment:
This should come back from mining_mode config using as_epic, other than the enabled check.